home-data/src/main/java/com/softserveinc/ita/homeproject/homedata/repository/InvitationRepository.java

```diff
@@ -0,0 +1,32 @@
+package com.softserveinc.ita.homeproject.homedata.repository;
+
+import java.time.LocalDateTime;
+import java.util.List;
+import java.util.Set;
+
+import org.springframework.data.jpa.repository.Modifying;
+import org.springframework.data.jpa.repository.Query;
+import org.springframework.data.repository.CrudRepository;
+import org.springframework.data.repository.query.Param;
+import org.springframework.stereotype.Repository;
+import org.springframework.transaction.annotation.Transactional;
+
+import com.softserveinc.ita.homeproject.homedata.entity.Invitation;
+
+@Repository
+public interface InvitationRepository extends CrudRepository<Invitation, Long> {
+
+    @Transactional
+    @Modifying
+    @Query("UPDATE Invitation i set i.status = true WHERE i.id in (:ids)")
```

Comment:
Seems to me it would be much understandable if we do it in the service layer.

Reply:
did it